tetris.js

```diff
@@ -308,5 +308,15 @@ module.exports = {
 		let coordinateDifference_X = coordinatesPivot.y - coordinatesElement.y
 		let coordinateDifference_Y = coordinatesPivot.x - coordinatesElement.x
 		return { x: coordinatesPivot.x - coordinateDifference_X, y: coordinatesPivot.y + coordinateDifference_Y }
+	},
+	rotatesArrayOfCoordinates: function(arrayCoordinates, coordinatePivot){
+		let localArrayCoordinates = []
+		if(arrayCoordinates.length === 0){
+			return arrayCoordinates
+		}	
+		for(let arrayElement of arrayCoordinates){
+			localArrayCoordinates.push(this.rotateElement(arrayElement,coordinatePivot[0]))	
```

Comment:
Pivot should be a single element, and not array

Reply:
Functiion getCoordinates return array, because it is an array

Reply:
This is not according to requirements, change parameters of rotatesArrayOfCoordinates